internal/cephfs/volume.go

```diff
@@ -102,23 +102,26 @@ func (vo *volumeOptions) getSubVolumeInfo(ctx context.Context, volID volumeID) (
 		Path:     info.Path,
 		Features: make([]string, len(info.Features)),
 	}
-	bc, ok := info.BytesQuota.(fsAdmin.ByteCount)
-	if !ok {
-		// If info.BytesQuota == Infinite (in case it is not set)
-		// or nil (in case the subvolume is in snapshot-retained state),
-		// just continue without returning quota information.
-		// TODO: make use of subvolume "state" attribute once
-		// https://github.com/ceph/go-ceph/issues/453 is fixed.
-		if !(info.BytesQuota == fsAdmin.Infinite || info.BytesQuota == nil) {
-			return nil, fmt.Errorf("subvolume %s has unsupported quota: %v", string(volID), info.BytesQuota)
+
+	// SubVolumeInfo returns all the attributes only when subvolume
+	// state is Complete.
+	if info.State == fsAdmin.StateComplete {
+		bc, ok := info.BytesQuota.(fsAdmin.ByteCount)
+		if !ok {
+			// If info.BytesQuota == Infinite (in case it is not set)
```

Comment:
is it the value `infinite` the default for ceph?

Reply:
yes